index.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<<<<<<< HEAD
+    <link rel="stylesheet" href="tribute.css" />
+    <title>tribute-page.freecodecamp.rocks/</title>
+  </head>
+  <body>
+    <main id="main">
+    <div class="container">
+      <div>
+        <div class="heading">
+          <h1 id="title">Dr. Norman Borlaug</h1>
+          <p>The man who saved a billion lives</p>
+        </div>
+      </div>
+      <div class="img-container">
+        <div class="image" id="img-div">
+          <img id="image"
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+            alt="Dr. Norman Borlaug"
+          />
+        </div>
+        <div class="img-text">
+          <caption id="img-caption">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+            on how to increase wheat yields - part of his life-long war on
+            hunger.
+          </caption>
+        </div>
+      </div>
+
+    </div>
+    <div class="mid-header" id="tribute-info">
+      <h2 >Here's a time line of Dr. Borlaug's life:</h2>
+    </div>
+    <div>
+      <div>
+        <ul>
+          <li>
+            <span class="date">1914</span> -
+            <span class="detail">Born in Cresco, Iowa</span>
+          </li>
+          <li>
+            <span class="date">1933</span> -
+            <span class="detail"
+              >Leaves his family's farm to attend the University of Minnesota,
+              thanks to a Depression era program known as the "National Youth
+              Administration"
+            </span>
```

Comment:
> Another `p` tag needed here

You did great for the first review about the use of `p` tag to ensure the accessibility 👏 . You also used polite language to suggest and gave a brief explanation `why`. But i suggest that the same review approach is used here. This will make the inline comment more meaningful to the project contributor(s) ☺️